horace_core/sqw/@Experiment/Experiment.m

```diff
@@ -334,25 +344,14 @@
                 avh.angdeg = sampl.angdeg;
             end
         end
-        %
-        function instr = get_unique_instruments(obj)
-            % compatibility fields with old binary file formats
-            instr = obj.instruments_.unique_objects;
-
-        end
-        %
-        function samp = get_unique_samples(obj)
-            % compatibility fields with old binary file formats
-
-            samp = obj.samples_.unique_objects;
-        end
-        %
+        %------------------------------------------------------------------
+        % REDUNDANT AND OLD FILE FORMAT COMPARTIBILITY METHOS
+        %------------------------------------------------------------------
         function head = get.header(obj)
             head = obj.convert_to_old_headers();
             head = [head{:}];
```

Comment:
```suggestion
            head = cell2mat(head);
```

Reply:
theoretically can be done, but is it worth to deploy equivalent implementations? I like the brackets more.